<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParentLab1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Задание 1</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Задание 1.3</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButtonSoliter">
            <property name="text">
             <string>Развернутый блокнот</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButtonKillSolitere">
            <property name="text">
             <string>Убить</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Задание 1.2</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButtonKillNotepad">
            <property name="text">
             <string>Убить</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="PushButtonCalc">
            <property name="text">
             <string>Калькулятор</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButtonKillCalc">
            <property name="text">
             <string>Убить</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButtonNotepad">
            <property name="text">
             <string>Блокнот</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButtonCmd">
            <property name="text">
             <string>Cmd</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButtonKillCmd">
            <property name="text">
             <string>Убить</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Задание 1.4 и 1.5 </string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButtonCreateChild">
            <property name="text">
             <string>Дочерний</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="sendMesageTOChild">
            <property name="text">
             <string>Отправить MESSAGE</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox_10">
         <property name="title">
          <string>Трио</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Открыть три</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Закрыть</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>Задание 3</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QLineEdit" name="SendTextlineEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTextBrowser" name="console"/>
    </item>
    <item row="2" column="2">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>Задание 2</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox_7">
      <property name="title">
       <string>Задание 4</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>Shared Memmory</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QPushButton" name="CreateSgaredMemmory">
            <property name="text">
             <string>Создать</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEditContentsOfSharedMemmory"/>
          </item>
          <item row="2" column="0">
           <widget class="QLineEdit" name="AddToSharedMemmoryEdit"/>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="WriteSharedMemmory">
            <property name="text">
             <string>Записать</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="ReadSharedMemmory">
            <property name="text">
             <string>Прочитать</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="pushButton_7">
            <property name="text">
             <string>Закрыть</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_9">
         <property name="title">
          <string>Shared File</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="CreateSharedFile">
            <property name="text">
             <string>Создать</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEdit_FromSharedFile"/>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="ReadSharedFile">
            <property name="text">
             <string>Прочитать</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLineEdit" name="lineEdit_ToSharedFile"/>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="WriteSharedFile">
            <property name="text">
             <string>Записать</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="CloseSharedFile">
            <property name="text">
             <string>Закрыть</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>533</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
